File: Python/Algorithm/programmers/alpha_string46.py
# [alpha_string46.py]
# 문자열 다루기 기본

# First
def solution1(s):
    try : 
        int(s)
        if len(s) == 4 or len(s) == 6 :
            return True 
        else:  
            return False
    except :
        return False

# Second
def solution2(s):
    return s.isdigit() and len(s) in (4, 6)

File: Java/Algorithm/programmers/alphaString46.java
// [alphaString46.java]
// 문자열 다루기 기본

class Solution {
  public boolean solution(String s) {
      boolean answer = true;
      // 1. 문자열의 길이가 4나 6인지 확인
      if (s.length() == 4 || s.length() == 6) {
        // 2. 각 문자가 숫자인지 확인
        for (int i = 0; i < s.length(); i++) {
          if(Character.isDigit(s.charAt(i))) {
              continue;
          } else {
              answer = false;
              break;
          }
        }
      }
      else {
          answer = false;
      }
      return answer;
    }
}